README.rst

```diff
@@ -89,7 +89,7 @@ Also note:
 - Documentation for the latest stable release is available
   |here1|_, and
 - Nightly version of the documentation that stays in sync with the
-  master branch is available |here2|_.
+  main branch is available |here2|_.
```

Comment:
That's... a proper name of the branch. Are we planning to rename our git master to main?

Reply:
In this context we not necessarily need to refer to the particular name but just to the default branch.

Reply:
Later on you have a direct link.

Reply:
use quotes/code then to indicate a name instead of quality/function?
```suggestion
  `master` branch is available |here2|_.
```

Reply:
And that is what I want to avoid. I think the `main branch` allows us to avoid naming the branch as `master`, and still, it is clear for the reader.